docs-ui/components/featureBadge.stories.js

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
```

Comment:
would be nice as it's own PR 😎

Reply:
no this is how i sneak things in

Reply:
yeah but I would have already approved this as a separate PR :trollface: